cli/translation-cli/src/main/kotlin/net/thunderbird/cli/translation/TranslationCli.kt

```diff
@@ -0,0 +1,73 @@
+package net.thunderbird.cli.translation
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.double
+import net.thunderbird.cli.translation.net.Language
+import net.thunderbird.cli.translation.net.WeblateClient
+
+const val TRANSLATED_THRESHOLD = 70.0
+
+class TranslationCli(
+    private val client: WeblateClient = WeblateClient(),
+    private val configurationsFormatter: ResourceConfigurationsFormatter = ResourceConfigurationsFormatter(),
+    private val supportedLanguagesFormatter: SupportedLanguagesFormatter = SupportedLanguagesFormatter(),
+) : CliktCommand(
+    name = "translation",
+    help = "Translation check CLI",
+) {
+    private val token: String by option(
+        help = "Weblate API token",
+    ).required()
+
+    private val threshold: Double by option(
+        help = "Threshold for translation completion",
+    ).double().default(TRANSLATED_THRESHOLD)
+
+    private val printAll: Boolean by option(
+        help = "Print code example",
+    ).flag()
+
+    override fun run() {
+        val languages = client.loadLanguages(token)
+        val filteredLanguages = filterByThreshold(languages, threshold)
+        val androidVersionCodes = mapToAndroidVersionCodes(filteredLanguages)
```

Comment:
Note: Using extension functions this could be written in a more functional style:

```kotlin
val androidLanguageCodes = client.loadLanguages(token)
    .filterByThreshold(threshold)
    .mapToAndroidLanguageCodes()
```

Reply:
Right, but having properties makes it easier to debug the script when needed.